src/megameklab/com/printing/PrintSmallUnitSheet.java

```diff
@@ -109,4 +113,29 @@ protected String getRecordSheetTitle() {
         // Not used by composite sheet
         return "";
     }
+
+    private void drawFluffImage() {
+        if (entities.size() > 1) {
+            for (int i = 1; i < entities.size(); i++) {
+                if (!entities.get(i).getChassis().equals(entities.get(0).getChassis())) {
```

Comment:
Wouldn't this be best compared the opposite way around?

Reply:
What difference does it make?